src/transformStateWithClones.js

```diff
@@ -65,7 +65,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
```

Comment:
Незрозуміло який результат, результат чого і тд
```suggestion
  const states = [];
```